spec/lib/krikri/harvesters/oai_harvester_spec.rb

```diff
@@ -403,25 +417,6 @@
       end
     end
 
```

Comment:
Could you comment on why the tests were moved out of the `#record_xml` describe block?